src/main/java/org/folio/rest/impl/VoucherNumberHelper.java

```diff
@@ -0,0 +1,66 @@
+package org.folio.rest.impl;
+
+import static org.folio.invoices.utils.ResourcePathResolver.VOUCHER_NUMBER_START;
+import static org.folio.invoices.utils.ResourcePathResolver.resourcesPath;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.invoices.utils.HelperUtils;
+import org.folio.rest.tools.client.interfaces.HttpClientInterface;
+
+import io.vertx.core.Context;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.logging.Logger;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+
+public class VoucherNumberHelper extends AbstractHelper {
```

Comment:
I believe that this logic can be moved to `VoucherHelper` because I do not think that there is going to be something else except get and set methods related to voucher number

Reply:
Done